src/engine/threaded_engine.h

```diff
@@ -293,6 +301,19 @@ class ThreadedEngine : public Engine {
     finished_cv_.notify_all();
   }
 
+  static int DefaultOMPThreadsPerWorker() {
+    int cores = std::thread::hardware_concurrency();
```

Comment:
physical core or logical core?

Reply:
I am changing in other branch to omp number of processors call.